resources/model/OpenStudio.idd

```diff
@@ -17905,6 +17906,60 @@ OS:Coil:Heating:Electric,
        \type object-list
        \object-list Node
 
+OS:Coil:Heating:Electric:MultiStage,
+       \memo Electric heating coil, multi-stage. If the coil is located directly in an air loop
+       \memo branch or outdoor air equipment list, then it is controlled on leaving air temperature
+       \memo and the Temperature Setpoint Node Name must be specified. If the coil is contained
+       \memo within another component such as an air terminal unit, zone HVAC equipment, or unitary
+       \memo system, then the coil is controlled by the parent component and the setpoint node name
+       \memo is not entered.
+       \extensible:1
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2 , \field Name
+       \type alpha
+       \required-field
+       \reference HeatingCoilsElectricMultiStage
+       \reference ConnectionObject
+  A3 , \field Availability Schedule
+       \note Availability schedule name for this system. Schedule value > 0 means the system is available.
+       \note If this field is blank, the system is always available.
+       \type object-list
+       \object-list ScheduleNames
+  A4 , \field Air Inlet Node
+       \type object-list
+       \required-field
+       \object-list ConnectionNames
+  A5 , \field Air Outlet Node
+       \type object-list
+       \required-field
+       \object-list ConnectionNames
+  A7;  \field Stage 1
+       \begin-extensible
```

Comment:
No `Temperature Setpoint Node Name` field?

Reply:
Cool cool cool, **None** of the E+ example files actually have a Coil:Heating:Electric:MultiStage with a Temperature Setpoint Node Name set.

https://github.com/NREL/EnergyPlus/blob/599c64df5584701dc1c2020f9667c5f7a720fb34/doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex#L1034-L1036

> **Field: Temperature Setpoint Node Name**
> If the coil is used in the air loop simulation directly on a branch and is temperature controlled using a Set Point Manager, then the node that is the control node needs to be specified here. If the coil is used in an air terminal unit or other parent object (ZoneHVAC: or AirloopHVAC:), the coil is controlled by the parent object and the temperature set point node name is not required. At present, the multistage electric heating coil does not model temperature setpoint control.